index.html

```diff
@@ -2,194 +2,225 @@
+          <p class="buttons">
+            <a class="btn btn-lg"
+               href="https://docs.bazel.build/install.html">
+               Get Bazel
+            </a>
+            <a class="btn btn-success btn-lg main-cta"
+               href="https://docs.bazel.build/getting-started.html">
+               Get Started
+            </a>
+          </p>
+        </div>
       </div>
     </div>
   </div>
-</div>
-<div class="landing-feature-2">
-  <div class="container">
-    <div class="row">
-      <div class="col-sm-12">
-        <h1>Why Bazel?</h1>
-        <p class="lead">Here's what makes Bazel great.</p>
-      </div>
-    </div>
-    <div class="row">
-      <div class="col-sm-4">
-        <h3>Speed</h3>
-        <p>With Bazel's optimized dependency analysis, advanced
-           caching and parallel execution of build actions, you get
-           fast, clean builds.
-        </p>
-      </div>
-      <div class="col-sm-4">
-        <h3>Scalability</h3>
-        <p>At Google, Bazel is used by tens of
-           thousands of engineers to build heavy-duty, mission-critical
-           infrastructure services as well as public web applications. But it's
-           great for smaller projects, too.
-        </p>
-      </div>
-      <div class="col-sm-4">
-        <h3>Flexibility</h3>
-        <p>A comprehensive, built-in set of rules lets you build software for
-           a wide variety of languages and platforms right out of the box.
-           Skylark, Bazel's extensible rule framework, allows you to develop
-           and share your own build rules for other languages and platforms.
-        </p>
-      </div>
-    </div>
-    <div class="row">
-      <div class="col-sm-4">
-        <h3>Correctness</h3>
-        <p>Bazel examines the content of source files in your
-           dependency graph, not just timestamps, to determine when to rebuild.
-           Because incremental builds are virtually always correct, it's not
-           necessary to run a clean build "just in case" to ensure correctness.
-        </p>
-      </div>
-      <div class="col-sm-4">
-        <h3>Reliability</h3>
-        <p>When you build software with Bazel, you're running the same code
-           that has been refined and tested over years of use in the
-           engineering environment at Google.
-        </p>
-      </div>
-      <div class="col-sm-4">
-        <h3>Repeatability</h3>
-        <p>Builds only use input files that are explicitly declared in the
-           build specification. On Linux, Bazel runs tools in a sandboxed
-           environment that contain only the minimum necessary files required.
-           Even tests are run in sandboxes, which ensures predictable,
-           environment-independent results.
-        </p>
+
+  <div class="landing-section summary">
+    <div class="container">
+      <div class="row">
+        <div class="col-md-8 col-sm-12">
```

Comment:
I left aligned the logo, which makes it aligned with the paragraph.
I removed the col-md-8.